batch/batch/front_end/templates/billing_projects.html

```diff
@@ -3,46 +3,50 @@
 {% block content %}
 <h1>Billing Projects</h1>
 <div class="searchbar-table">
-  <table class="data-table" id="batches">
+  <table class="grouped-data-table" id="billing-projects">
     <thead>
       <tr>
-        <th>Billing Project</th>
+        <th colspan="2">Billing Project</th>
         <th>User</th>
         <th></th>
       </tr>
     </thead>
-    <tbody>
       {% for billing_project in billing_projects %}
+    <tbody>
       <tr>
-        <td>{{ billing_project['billing_project'] }}</td>
-        <td></td>
-        <td></td>
-      </tr>
+        <td rowspan="{{ billing_project.size + 1 }}">
```

Comment:
Can you add a picture of what this looks like? My CSS/HTML knowledge is pretty limited.

Reply:
<img width="627" alt="Screen Shot 2020-10-07 at 4 53 07 PM" src="https://user-images.githubusercontent.com/19789871/95386697-ae3ce680-08bd-11eb-941c-4c6567a1de3e.png">

Reply:
(the part that I forgot to illustrate here is what happens if there are users in a closed billing project; I just display them on the same line as a comma-separated list.)

Reply:
Update: I'm tweaking this to combine the header cells for the "User" and <blank> (add/remove button) columns like I did for "Billing Project" and also to move the "Add" button in the last row under the "Close" button for the billing projects. I'll post a screenshot once it's done dev deploying.

Reply:
<img width="625" alt="Screen Shot 2020-10-08 at 12 40 26 PM" src="https://user-images.githubusercontent.com/19789871/95488598-bea79c80-0963-11eb-9494-f3513966fb66.png">

Reply:
I'm not sure we need the list of users.

Reply:
I'll take it out.

Reply:
This is really cool! I love this UI innovation.